src/coreclr/tools/aot/ILCompiler.Reflection.ReadyToRun/ReadyToRunReader.cs

```diff
@@ -1400,7 +1400,7 @@ private void FindOwnerCompositeExecutable()
                 {
                     int oceOffset = GetOffset(section.RelativeVirtualAddress);
                     string ownerCompositeExecutable = Encoding.UTF8.GetString(Image, oceOffset, section.Size - 1); // exclude the zero terminator
-                    _ownerCompositeExecutable = ownerCompositeExecutable.ToEscapedString().Replace("\"", ""); // remove double quotes
+                    _ownerCompositeExecutable = ownerCompositeExecutable.ToEscapedString(false);
```

Comment:
Please consider using named argument syntax (<code>ToEscapedString(placeQuotes: false)</code>) to make the code easier to read.

Reply:
Yea it's easier to read indeed!

Reply:
Frankly speaking, are we aware of any caller using the variant with the double quotes? I saw only this one so I'm wondering whether it might be actually easier to just drop the double quotes from <code>ToEscapedString</code> altogether. @cshung might have better understanding of the exact data flow here.

Reply:
I think the right thing to do is to return the raw string here and apply the `ToEscapeString()` only in the R2RDump case when we print things out.

The `ToEscapeString` was introduced by @AntonLapounov back in 2020 in https://github.com/dotnet/runtime/pull/36935 to avoid crashing when some string contains unpaired surrogates. 

While there is nothing we could do with user strings, I don't believe we should allow unpaired surrogates in the executable name. I think we should fail there instead.